Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -3857,4 +3850,40 @@ Vector<JS::NonnullGCPtr<Element>> Document::elements_from_point(double x, double
     return sequence;
 }
 
+// https://w3c.github.io/editing/docs/execCommand/#execcommand()
+bool Document::exec_command(String, bool, String)
+{
+    return false;
+}
+
+// https://w3c.github.io/editing/docs/execCommand/#queryCommandEnabled()
```

Comment:
Looks good, but this URL and the ones that follow don't work - the anchors should all be lowercase, e.g.:
`https://w3c.github.io/editing/docs/execCommand/#querycommandenabled()`

Reply:
😬 that's what I get for letting CoPilot generate the URLs instead of copying them from the spec